django-stubs/db/backends/dummy/features.pyi

```diff
@@ -1,5 +1,5 @@
 from django.db.backends.base.features import BaseDatabaseFeatures
 
 class DummyDatabaseFeatures(BaseDatabaseFeatures):
-    supports_transactions: bool
+    supports_transactions: bool  # type: ignore[assignment]
```

Comment:
Can you please explain this `ignore`? I don't get it :(

Reply:
This is solved by the `cached_property` changes too so I'll remove this from the MR. The error is the following:
> django-stubs/db/backends/postgresql/features.pyi:23: error: Incompatible types in assignment (expression has type "bool", base class "BaseDatabaseFeatures" defined the type as "Callable[[BaseDatabaseFeatures], bool]")  [assignment]